Examples/Tests/embedded_circle/inputs_2d

```diff
@@ -0,0 +1,80 @@
+max_step = 1
+warpx.const_dt = 3.99e-13
+warpx.do_electrostatic = labframe
+warpx.self_fields_required_precision = 1e-06
+warpx.eb_implicit_function = -((x-0.00005)**2+(z-0.00005)**2-1e-05**2)
+warpx.eb_potential(x,y,z,t) = -10
+warpx.self_fields_absolute_tolerance = 0.02
+amr.n_cell = 128 128
+amr.max_grid_size = 32
+amr.max_level = 0
+geometry.coord_sys = 0
+geometry.prob_lo = 0.0 0.0
+geometry.prob_hi = 0.0001 0.0001
+boundary.field_lo = pec periodic
+boundary.field_hi = pec periodic
+boundary.particle_lo = absorbing periodic
+boundary.particle_hi = absorbing periodic
+algo.particle_shape = 1
+particles.species_names = electrons ar_ions
+electrons.mass = m_e
+electrons.charge = -q_e
+electrons.injection_style = nuniformpercell
+electrons.initialize_self_fields = 1
+electrons.num_particles_per_cell_each_dim = 2 2
+electrons.density = 10.2e16
+electrons.momentum_distribution_type = gaussian
+electrons.profile = constant
+electrons.ux_m = 0.0
+electrons.uy_m = 0.0
+electrons.uz_m = 0.0
+electrons.ux_th = 0.0022492462021094224
+electrons.uy_th = 0.0022492462021094224
+electrons.uz_th = 0.0022492462021094224
+electrons.save_particles_at_zlo = 1
+electrons.save_particles_at_zhi = 1
+electrons.save_particles_at_eb = 1
+ar_ions.mass = 6.633520326333601e-26
+ar_ions.charge = q_e
```

Comment:
Shall we add argon to the pre-defines particle types? :)
(as a follow-up PR, I mean)

Reply:
That sounds like a good idea! We'll open up another PR for that.